src/CLITests/ProcessHelperUnitTests.cs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using AxeWindowsCLI;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.Diagnostics;
+
+namespace CLITests
+{
+    [TestClass]
+    public class ProcessHelperUnitTests
```

Comment:
I think this class is missing a successful check of GetProcessById.

Reply:
Added